pandas/core/api.py

```diff
@@ -4,7 +4,8 @@
 
 import numpy as np
 
-from pandas.core.algorithms import factorize, match, unique, value_counts
+from pandas.core.algorithms import (checked_add_with_arr, factorize, match,
```

Comment:
NO! this is not a top-level function, nor should be actually exposed to the user